app/main.py

```diff
@@ -1,8 +1,51 @@
+def limitation(value: float,
+               min_value: float,
+               max_value: float) -> float:
+    return max(min(value, max_value), min_value)
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = int(limitation(comfort_class, 1, 7))
+        self.clean_mark = int(limitation(clean_mark, 1, 10))
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = limitation(
+            distance_from_city_center, 1.0, 10.0)
+        self.clean_power = int(limitation(clean_power, 1, 10))
+        self.average_rating = round(limitation(average_rating, 1.0, 5.0), 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = 0.0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return income
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round((car.comfort_class
+                      * (self.clean_power - car.clean_mark)
+                      * self.average_rating
+                      / self.distance_from_city_center), 1)
```

Comment:
The best approach of placing calculations is to move them on the next line after variable like
```
var = (
    var1 + var2
    * var3
    / var4
)
```